src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs

```diff
@@ -64,7 +63,6 @@ public void ValidFileOptions(FileOptions option)
             }
         }
 
-        [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
```

Comment:
This didn't just disable for Nano... it disabled the tests entirely.  You need to replace the ConditionalTheory attributes with Theory.